common/components/deploy_model/README.md

```diff
@@ -0,0 +1,152 @@
+# Model Deployment Component
+This component can be used in [azure machine learning pipelines](https://learn.microsoft.com/en-us/azure/machine-learning/concept-ml-pipelines?view=azureml-api-2) to [deploy](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-deploy-online-endpoints?view=azureml-api-2&tabs=azure-cli) the model in [workspace](https://learn.microsoft.com/en-us/azure/machine-learning/concept-workspace?view=azureml-api-2).
+
+The components can be seen here ![as shown in the figure]
+
+# 1. Inputs
```

Comment:
@HrishikeshGeedMS, was this generated using @ManojBableshwar's script? @lykelly19 is working on publishing documentation like this on Microsoft's documentation site instead of here.

Reply:
@elesel , No this was not created using @ManojBableshwar's script. I took reference from this PR - https://github.com/Azure/azureml-assets/pull/562 .